code/modules/reqs/supply.dm

```diff
@@ -633,7 +633,6 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 	var/datum/supply_beacon/beacon_datum
 
 /obj/item/storage/backpack/marine/radiopack/attack_hand_alternate(mob/living/user)
-	. = ..()
 	if(.)
```

Comment:
then . will always be false

Reply:
Cleaned it up. Also I added an extra examine action to tell the functions of the item. Changes log looks messy due to me adding stuff to the lines above.